Makefile

```diff
@@ -16,17 +16,19 @@ PKG=github.com/hashicorp/vault-csi-provider/internal/version
 LDFLAGS?="-X '$(PKG).BuildVersion=$(VERSION)' \
 	-X '$(PKG).BuildDate=$(BUILD_DATE)' \
 	-X '$(PKG).GoVersion=$(shell go version)'"
-CSI_DRIVER_VERSION=1.4.6
+CSI_DRIVER_VERSION=1.4.8
 VAULT_HELM_VERSION=0.29.1
-VAULT_VERSION=1.18.1
+VAULT_VERSION=1.19.0
 GOLANGCI_LINT_FORMAT?=colored-line-number
 
 VAULT_VERSION_ARGS=--set server.image.tag=$(VAULT_VERSION) --set csi.agent.image.tag=$(VAULT_VERSION)
 ifdef VAULT_LICENSE
 	VAULT_VERSION_ARGS=--set server.image.repository=docker.mirror.hashicorp.services/hashicorp/vault-enterprise \
 		--set server.image.tag=$(VAULT_VERSION)-ent \
 		--set server.enterpriseLicense.secretName=vault-ent-license \
-		--set csi.agent.image.tag=$(VAULT_VERSION)
+		--set csi.agent.image.repository=docker.mirror.hashicorp.services/hashicorp/vault-enterprise \
+		--set csi.agent.image.tag=$(VAULT_VERSION)-ent
```

Comment:
Since this was previously deploying the CE Vault image for Agent, this limited the Vault versions we could test to CE releases. For something like 1.16 which is now LTS at 1.16.17+ent, we were stuck at 1.16.3. Since the vault-enterprise image can be used in Agent mode, this allows us to test the latest ENT Vault releases.